### ISSUE TITLE
Create quizzes

### ISSUE THREAD

**Issue by lukecookssw** [SSWConsulting/SSW.Rewards#353]
@wicksipedia @william-liebenberg @matt-goldman @lukecookssw @adamcogan

As per our conversation, the following quizzes need to be created:

## Quizzes
- [ ] Angular quiz
- [ ] MAUI quiz
- [ ] Blazor quiz
- [ ] React quiz
- [ ] Javascript quiz
- [ ] CSS quiz
- [ ] .NET quiz
- [ ] GitHub quiz
- [ ] Azure quiz
- [ ] Power Platform quiz
- [ ] Scrum quiz
- [ ] Consulting quiz

## Acceptance Criteria
- [ ] Each quiz has **10 questions**, with each question having **4 possible answers** with **1 correct answer.**
- [ ] Each quiz has a unique achievement linked to it.
- [ ] Quizzes are stored **in the SSW.Rewards Teams channel** ready to be inserted into the chosen quiz platform.

**Comment by AntPolkanov** [comment]
This is my absolute favorite, and it has huge potential 🚀 I am looking forward to quizzes on particular topics let's say under the .NET umbrella: 

- LINQ
- Collections
- Async/Await
- DI
- New features of C# 12
- etc

Each of these topics can be split into levels of difficulty - basic, intermediate, advanced. People can use the quizzes to check the depth of their knowledge.

BTW, I am happy to prepare the questions🙂

**Comment by JimmyChenSSW** [comment]
Even more, we need annual quizzes championship!!

**Comment by PennyWalker** [comment]
I really love this idea - Commbank do a FriYay quiz every week and it looks super fun - I think it'll be great for engagement and for finding potential devs to hire

**Comment by Marxoz** [comment]
Yes, this will be great. It will be good to add a rule to the answer, even if they get it right. Just to promote our rules a bit more

**Comment by sylhuang** [comment]
I love this idea and I think the knowledge we collected overtime could be easily transfered into some other purposes (e.g. educational app for profit)

**Comment by AntPolkanov** [comment]
Before continuing check email with subject "Check how many quizzes were solved".

**Comment by AntPolkanov** [comment]
Since we introduced the quizzes feature:
- 284 quizzes were solved
- 72 users solved at least one quiz; 18 of them are SSW employees, 54 are external. Note that there might be some overlap here as some employees have Rewards profiles registered on their personal email, but there is no way for us to figure it out.
